src/core/jsx.ts

```diff
@@ -17,13 +20,30 @@ export class JsxRenderer {
     return JsxRenderer.transformJsxToHtml(jsx).outerHTML;
   }
 
+  private static addElementEventListener(attribute: string, jsx: Jsx, element: HTMLElement) {
+    const event = attribute.split('on')[1] as EventTypes;
+    const func = jsx.attributes[attribute] as unknown as (event: Event | null) => any;
+    const id = element.attributes['id'] ? element.attributes['id'].nodeValue : Guid.NewGuid();
+    element.setAttribute('id', id);
+
+    setTimeout(() => {
+      new Command(() => {
```

Comment:
If you set something like a mouseover event and swipe the mouse around like crazy ;) there's some potential for the element to be removed (by virtue of re-render) right before this call to add a listener completes.

The command will log any thrown errors as warnings when in dev mode.

Reply:
At first I through an Elvis operator on there like this: `element?.addEventListener(event, func);`, but then you have another branch to test.